src/components/reservations/ReservationReleasePDF.tsx

```diff
@@ -0,0 +1,93 @@
+import React, { useState } from 'react';
+import { Button, IconDocument, IconDownload } from 'hds-react';
+import { useTranslation } from 'react-i18next';
+
+import { useGetCustomerByIdQuery } from '../../redux/services/api';
+import { useDownloadFile } from '../../utils/useDownloadFile';
+import { useFileDownloadApi } from '../../utils/useFileDownloadApi';
+import { toast } from '../common/toast/ToastManager';
+
+import styles from './ReservationReleasePDF.module.scss';
+
+const T_PATH = 'components.reservations.ReservationReleasePDF';
+
+interface Props {
+  reservationId: number;
+  customerId: number;
```

Comment:
I would prefer having the original type here so that if something gets changed, these won't necessarily break.

```
reservationId: ApartmentReservation['id'];
customerId: Customer['id'];
```